Objects/dictobject.c

```diff
@@ -3929,6 +3929,30 @@ static PyMethodDef mapp_methods[] = {
     {NULL,              NULL}   /* sentinel */
 };
 
+
+// return 1 if the module `key` in `mp` is not loaded (a lazy key)
+// return 0 if the module `key` in `mp` is loaded
+// return -1 if existing an error
+int
+PyDict_IsLazyKey(PyObject *mp, PyObject *key)
+{
+    PyObject *value = _PyDict_GetItemKeepLazy(mp, key);
+
+    // error
+    if (value == NULL) {
+        return -1;
+    }
```

Comment:
I think it would be cleaner to throw an exception instead of returning `-1` (e.g. `KeyError`), which would allow the Python wrapper to return `bool` instead if `int`